api/src/main/java/org/openmrs/api/db/hibernate/envers/OpenMRSRevisionEntity.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.api.db.hibernate.envers;
+
+import org.hibernate.envers.DefaultRevisionEntity;
+import org.hibernate.envers.RevisionEntity;
+import org.joda.time.DateTime;
+
+import javax.persistence.Entity;
+import java.util.Date;
+
+@RevisionEntity(OpenMRSRevisionEntityListener.class)
+@Entity
+public class OpenMRSRevisionEntity extends DefaultRevisionEntity {
```

Comment:
Can you include some description of why we need this class?

Reply:
I added a description.